packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts

```diff
@@ -93,6 +93,11 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * @default false
```

Comment:
Please add a change request json file. Instructions:

1. make sure you have an upstream remote to officedev url (I call mine "upstream")

`npm run change --b upstream/master`

This change is a "minor" package bump.

For comment, these show up in release nows. Please follow the covention `ComponentName: change description`.

`BasePicker: added `itemLimit` property (etc).`